docs/api-guide/serializers.md

```diff
@@ -800,6 +800,9 @@ Here's an example of how you might choose to implement multiple updates:
             return ret
 
     class BookSerializer(serializers.Serializer):
+        # You must explicitly mark the primary key as editable.
```

Comment:
Let's tweak this slightly.

# We need to identify elements in the list using their primary key,
# so use a writable field here, rather than the default which would be read-only.